packages/checkbox/src/checkbox.vue

```diff
@@ -64,19 +64,19 @@ import {
   getCurrentInstance,
   watch,
   onMounted,
-  // nextTick,
+  PropType,
 } from 'vue'
 import { useCheckbox } from './useCheckbox'
 
 export default defineComponent({
   name: 'ElCheckbox',
   props: {
     modelValue: {
-      type: [Object, Boolean],
+      type: [Object, Boolean] as PropType<Record<string, unknown> | boolean>,
```

Comment:
Is there a way to model this `modelValue`?

Reply:
This prop is usually primitive type. Do you have any suggestions？